### PR TITLE
[JENKINS-43863] Improve PCT support

### PR DESCRIPTION
**Pull request by varyvol**
[JENKINS-43863](https://issues.jenkins-ci.org/browse/JENKINS-43863)

- Bump Jenkins version and set appropriate java.level.
- Include needed dependencies.

@reviewbybees esp. @andresrc @jglick @petehayes @mambu

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,8 +17,8 @@
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Dashboard+View</url>
 
   <properties>
-    <jenkins.version>1.554.1</jenkins.version>
-    <findbugs.failOnError>false</findbugs.failOnError> <!-- TODO fix -->
+    <jenkins.version>1.580.1</jenkins.version>
```
Comment (varyvol) [inline pom.xml]: Given this is still a quite old version, I don't think the bumping is a problem.

**Comment by varyvol** [comment]
@olivergondza could you review and merge this if appropriate?

**Comment by varyvol** [comment]
@olivergondza would it be possible to release a new version including this PR and [PR #49](https://github.com/jenkinsci/dashboard-view-plugin/pull/49)?

**Comment by olivergondza** [comment]
@varyvol, done. Please consider becoming a maintainer yourself.

**Pull request merged**